src/config/constants/farms.ts

```diff
@@ -38,6 +38,16 @@ const farms: FarmConfig[] = [
   /**
    * V3 by order of release (some may be out of PID order due to multiplier boost)
    */
+  {
+    pid: 433,
+    lpSymbol: 'IF-BNB LP',
```

Comment:
lpSymbol is wrong, it should be IF-BUSD
```suggestion
    lpSymbol: 'IF-BUSD LP',
```